orchagent/main.cpp

```diff
@@ -76,6 +77,7 @@ void usage()
     cout << "    -m MAC: set switch MAC address" << endl;
     cout << "    -i INST_ID: set the ASIC instance_id in multi-asic platform" << endl;
     cout << "    -s: enable synchronous mode" << endl;
```

Comment:
Could you also add a message that this is being deprecated as well?

Reply:
done